Docs/sphinx_documentation/source/BuildingAMReX.rst

```diff
@@ -477,6 +479,8 @@ below.
    +------------------------------+-------------------------------------------------+-------------+-----------------+
    | ENABLE_MEM_PROFILE           |  Build with memory-profiling support            | NO          | YES, NO         |
    +------------------------------+-------------------------------------------------+-------------+-----------------+
+   | ENABLE_MPI_THREAD_MULTIPLE   |  Multiple threads can make MPI concurrently     | NO          | YES, NO         |
```

Comment:
Threads don't "make" MPI concurrently, correct? They runs distinct MPI calls concurrently.

Reply:
Or "Build with support for concurrent MPI calls on multiple threads"?

Reply:
argh, word "calls" is missing :D

Reply:
```suggestion
   | ENABLE_MPI_THREAD_MULTIPLE   |  Concurrent MPI calls from multiple threads     | NO          | YES, NO         |
```